src/main/java/com/jcabi/github/Issues.java

```diff
@@ -85,4 +87,116 @@ Issue create(
     Iterable<Issue> iterate(@NotNull(message = "map of params can't be NULL")
         Map<String, String> params);
 
+    /**
+     * Search for issues within the given repository.
+     *
+     * @param sort The sort field
+     * @param direction The sort direction
+     * @param qualifiers The search qualifier
+     * @return Issues
+     * @throws java.io.IOException If there is any I/O problem
+     * @since 0.22.0
+     * @see <a href="https://developer.github.com/v3/issues/#list-issues-for-a-repository">List issues for a repository</a>
+     */
+    @NotNull(message = "Iterable of issues is never NULL")
+    Iterable<Issue> search(
+        @NotNull(message = "Sort field can't be NULL") Sort sort,
+        @NotNull(message = "Sort direction can't be NULL")
+        Search.Order direction,
+        @NotNull(message = "Search qualifiers can't be NULL")
+        EnumMap<Qualifier, String> qualifiers)
+        throws IOException;
+
+    enum Qualifier {
+        /**
+         * Filter issues based on which milestone they are assigned to.
+         * "none" means no assigned milestone. "*" means any milestone.
+         */
+        MILESTONE("milestone"),
+        /**
+         * Filter issues based on whether they're open or closed.
+         */
+        STATE("state"),
+        /**
+         * Finds issues that are assigned to a certain user.
+         * "none" means no assigned user. "*" means assigned to any user.
+         */
+        ASSIGNEE("assignee"),
+        /**
+         * Finds issues created by a certain user.
+         */
+        CREATOR("creator"),
+        /**
+         * Finds issues that mention a certain user.
+         */
+        MENTIONED("mentioned"),
+        /**
+         * Filters issues based on their labels,
+         * as a comma-separated list of label names.
+         * An issue must have all of the labels in the list in order to
+         * appear in the search results.
+         */
+        LABELS("labels"),
+        /**
+         * Filters issues based on date last updated (as an ISO 8601 timestamp).
+         */
+        SINCE("since");
+
+        /**
+         * Search qualifier.
+         */
+        private final transient String qualifier;
+
+        /**
+         * Ctor.
+         * @param key Search qualifier
+         */
+        Qualifier(final String key) {
+            this.qualifier = key;
+        }
+
+        /**
+         * Get search qualifier.
+         * @return String
+         */
+        public String getQualifier() {
```

Comment:
usually we avoid using of getters / setters, let's rename this method (and maybe the field as well) as described [here](http://www.yegor256.com/2014/09/16/getters-and-setters-are-evil.html).